src/main/java/magicbees/main/Config.java

```diff
@@ -226,6 +228,15 @@ public void setupItems() {
         setupMiscForestryItemHooks();
     }
 
+    public void setupThaumicEnergistics() {
+        if (isThaumicEnergisticsLoaded) {
+            try {
+                Class c = BlockApimancersDrainer.drainer;
```

Comment:
You should register TileEntity class (read method name and check code in my previous comment). And `try`/`catch` block isn't needed here.

Reply:
> You should register TileEntity class (read method name and check code in my previous comment).
Yes, `BlockApimancersDrainer.drainer` is the used TileEntity as defined on L#522 (see below):
```java
            if (isGTNHCoreModLoaded) BlockApimancersDrainer.drainer = TileEntityApimancersDrainerGT.class;
            apimancersDrainer = new BlockApimancersDrainer();
            GameRegistry.registerBlock(apimancersDrainer, "apimancersDrainer");
            GameRegistry.registerTileEntity(BlockApimancersDrainer.drainer, "apimancersDrainer");
```

Reply:
> And try/catch block isn't needed here.

This follows the provided example from Gadomancy, and IntelliJs suggestion as apparently `transportPermissions` can be null.